charts/tidb-operator/templates/scripts/_create_cert.sh

```diff
@@ -0,0 +1,159 @@
+set -e
```

Comment:
Currently I choose to use script to generate ca cert, secrets and csr to avoid repeating cert and secret control as https://github.com/pingcap/tidb-operator/pull/782 is working on it either. I will generate all the above things in code after https://github.com/pingcap/tidb-operator/pull/782 is merged.